frontend/src/App.js

```diff
@@ -1,5 +1,10 @@
 import { useState, useEffect } from "react"
-import { Routes, Route, useNavigate } from "react-router-dom"
+import {
+  Routes,
+  Route,
+  useNavigate,
+  useSearchParams,
```

Comment:
This is unused in the app file and should be removed

Reply:
done